baselines/webworker.generated.d.ts

```diff
@@ -1077,8 +1157,8 @@ declare var PushManager: {
 interface PushMessageData {
     arrayBuffer(): ArrayBuffer;
     blob(): Blob;
-    json(): JSON;
-    text(): USVString;
+    json(): any;
```

Comment:
We should avoid using `any` in the lib, right? Meaning this should probably return `object | number | string | Function` or something?

Reply:
> We should avoid using any in the lib, right?

well.. i do nto know.. some of these are from the spec, so if not an accurate type, i would not want to mess up with them..

Reply:
I'm saying we should use the union of all possible base types instead of `any`, that way no `any`'s seep into user code from the lib. What was `JSON` defined as before, anyway?

Reply:
This one specifically came from the webidl file. i would argue it is better than `object | string ..` similar to `JSON.parse()`